assets/src/edit-story/components/richText/karma/inlineSelection.karma.js

```diff
@@ -297,10 +297,10 @@ describe('CUJ: Creator can Add and Write Text: Select an individual word to edit
         // Exit edit-mode
         await data.fixture.events.keyboard.press('Escape');
 
-        // Assume text content to be correctly formatted
-        const actual = getTextContent();
-        const expected = `<span style="font-weight: 900">F</span><span style="font-weight: 700">i</span><span style="font-weight: 700">l</span>l in some text`;
-        expect(actual).toBe(expected);
+        // Assume visual match (actual HTML might vary, but visual must be similar)
```

Comment:
Curious why would the actual HTML vary if the steps taken are the same. Is it related to the time it needs to take effect? E.g. would adding a timeout always ensure the same result?

Reply:
I have no idea why. It might be because it's run on slightly different hardware or something like that. No idea - I just see that `draftjs` return different results on the same input.